test/eChronicler/journal_entries_test.exs

```diff
@@ -10,4 +10,21 @@ defmodule EChronicler.JournalEntriesTest do
     assert JournalEntries.list_journal_entries() == [journal_entry_one, journal_entry_two]
   end
 
+  test "reverse_chron_list_journal_entries/0 lists journal entries in reverse chronological order" do
+    {:ok, journal_entry_1} = JournalEntries.create_journal_entry(%{author: "Tiny", title: "1", entry: "1"})
+    {:ok, journal_entry_2} = JournalEntries.create_journal_entry(%{author: "Tiny", title: "2", entry: "2"})
+    {:ok, journal_entry_3} = JournalEntries.create_journal_entry(%{author: "Tino", title: "3", entry: "3"})
+    {:ok, journal_entry_4} = JournalEntries.create_journal_entry(%{author: "Tino", title: "4", entry: "4"})
+    {:ok, journal_entry_5} = JournalEntries.create_journal_entry(%{author: "Tina", title: "5", entry: "5"})
+    assert JournalEntries.reverse_chron_list_journal_entries() == [journal_entry_5, journal_entry_4, journal_entry_3, journal_entry_2, journal_entry_1]
+  end
+
+  test "truncate_journal_entry/1 truncates entry" do
+    assert JournalEntries.truncate_journal_entry("Time is an illusion. Teatime doubly so. And time for tennis? Forget about it.") == "Time is an illusion. Teatime doubly so. And time fo..."
+  end
+
+  test "truncate_journal_entry/1 adds elipse after short entry" do
+    assert JournalEntries.truncate_journal_entry("Test") == "Test..."
+  end
+
```

Comment:
You might want to test what happens when the journal entry text is just an empty string.

Reply:
Done! I added a new case.